client/components/QuestionsAndAnswers/QuestionItem.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render, screen} from '@testing-library/react';
```

Comment:
@jerry-jma consider adding a space after "screen" or removing the space before "render"